fmralign/alignment_methods.py

```diff
@@ -465,3 +467,239 @@ def fit(self, X, Y):
     def transform(self, X):
         """Transform X using optimal coupling computed during fit."""
         return X.dot(self.R)
+
+
+class IndividualizedNeuralTuning(Alignment):
+    """
+    Method of alignment based on the Individualized Neural Tuning model, by Feilong Ma et al. (2023).
+    It works on 4D fMRI data, and is based on the assumption that the neural response to a stimulus is shared across subjects.
+    It uses searchlight/parcelation alignment to denoise the data, and then computes the stimulus response matrix.
+    See article : https://doi.org/10.1162/imag_a_00032
+    """
+
+    def __init__(
+        self,
+        template="pca",
+        decomp_method=None,
+        n_components=None,
+        alignment_method="searchlight",
+        n_jobs=1,
+    ):
+        """
+        Initialize the IndividualizedNeuralTuning object.
+        Parameters:
+        --------
+
+        - template (str): The type of template to use for alignment. Default is "pca".
```

Comment:
Additionally, you should indicate what are the possible alternatives for this argument.